integration/system/stats_test.go

```diff
@@ -202,6 +193,37 @@ func getNetworkStats(c *testing.T, id string) map[string]types.NetworkStats {
 	return st.Networks
 }
 
+func RuntimeIsWindowsContainerd() bool {
+	return os.Getenv("DOCKER_WINDOWS_CONTAINERD_RUNTIME") == "1"
+}
+
+func TestAPIStatsNetworkStatsVersioning(t *testing.T) {
+	// Windows doesn't support API versions less than 1.25, so no point testing 1.17 .. 1.21
+	skip.If(t, !testEnv.IsLocalDaemon())
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
```

Comment:
All not urgent (existing things), but this is also an odd check;

- if it's really about the API version only, we could skip based on the daemon's supported API version instead.
- Not sure why we have the `local daemon` requirement.
- For the "!= linux" part; I'm guessing these properties are Linux-only; in which case we could add a comment in that check to describe that.

Reply:
Indeed. However, I think in general we can look past these since it looks like this test as a whole might not make since after we we fully drop API < 1.24.